pyquil/api/_quantum_computer.py

```diff
@@ -1146,7 +1147,7 @@ def _consolidate_symmetrization_outputs(
 
 
 def _measure_bitstrings(
-    qc: QuantumComputer, programs: List[Program], meas_qubits: List[int], num_shots: int = 600
+    qc: QuantumComputer, programs: List[Program], meas_qubits: List[int], num_shots: int = 600, memory_map: Optional[Mapping[str, Sequence[Union[int, float]]]] = None
```

Comment:
The linter is complaining about an overlong line here. You can run `make format` to run the black formatter, which should fix it.